_topic_map.yml

```diff
@@ -1041,11 +1041,13 @@ Topics:
     File: planning-migration-3-to-4
   - Name: Migrating OpenShift Container Platform 3.7 to 4.2
     File: migrating-openshift-3-to-4
-- Name: Migrating OpenShift Container Platform 4.1 to 4.2
+- Name: Migrating OpenShift Container Platform 4 to 4
   Dir: migrating-4-4
   Topics:
   - Name: Migrating OpenShift Container Platform 4.1 to 4.2
-    File: migrating-openshift-4-to-4
+    File: migrating-openshift-4_1-to-4
+  - Name: Migrating OpenShift Container Platform 4.2 to 4.2
```

Comment:
This seems really confusing titled as is, from 4.2 to 4.2. You said in slack that this was for migrating OCP workload without changing versions. Suggest to make that clearer in this title. 

Migrating workloads between OpenShift Container Platform 4.2 clusters

Reply:
Agree.